heartbeat/sender/bootstrapSender.go

```diff
@@ -0,0 +1,77 @@
+package sender
+
+import (
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go-core/core"
+	"github.com/ElrondNetwork/elrond-go-core/marshal"
+	crypto "github.com/ElrondNetwork/elrond-go-crypto"
+	"github.com/ElrondNetwork/elrond-go/heartbeat"
+	"github.com/ElrondNetwork/elrond-go/heartbeat/sender/disabled"
+)
+
+// ArgBootstrapSender represents the arguments for the bootstrap bootstrapSender
+type ArgBootstrapSender struct {
+	Messenger                          heartbeat.P2PMessenger
+	Marshaller                         marshal.Marshalizer
+	HeartbeatTopic                     string
+	HeartbeatTimeBetweenSends          time.Duration
+	HeartbeatTimeBetweenSendsWhenError time.Duration
+	HeartbeatThresholdBetweenSends     float64
+	VersionNumber                      string
+	NodeDisplayName                    string
+	Identity                           string
+	PeerSubType                        core.P2PPeerSubType
+	CurrentBlockProvider               heartbeat.CurrentBlockProvider
+	PrivateKey                         crypto.PrivateKey
+	RedundancyHandler                  heartbeat.NodeRedundancyHandler
+	PeerTypeProvider                   heartbeat.PeerTypeProviderHandler
+}
+
+// bootstrapSender defines the component which sends heartbeat messages during bootstrap
+type bootstrapSender struct {
+	heartbeatSender *heartbeatSender
```

Comment:
so heartbeat sender will be used in the next PRs

Reply:
correct..it is already started, but in the next PR a new component will be injected so it would be able to send the trie sync statistics